### PR TITLE
Add /faucet page

### PR DESCRIPTION
**Pull request by DoubleOTheven**
Code changes for [this Git issue](https://github.com/paritytech/ink-docs/issues/157)

* Add `/faucet` path with prettified faucet
* Update `/testnet` overview docs to link to the new faucet page
* Responsive layouts for mobile, tablet, and desktop
* Add Montserrat font for `/faucet` page

The React component for Recaptcha is not respecting the theme prop so the color is not turning to black as specified in the illustration [here](https://github.com/paritytech/ink-docs/issues/157). I personally think it looks good with the clean white, but can change this if it is a deal breaker.

Desktop
-------
Screen Shot 2023-02-10 at 7 33 10 PM

Screen Shot 2023-02-10 at 7 35 16 PM

iPhone 11
-------
Screen Shot 2023-02-10 at 7 33 54 PM

Screen Shot 2023-02-10 at 7 34 56 PM

Updated Testnet Docs
---------------
Screen Shot 2023-02-10 at 7 37 59 PM

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -11,26 +11,122 @@
   --ifm-color-primary-dark: #5a007e;
   --ifm-color-primary-darker: #5a007e;
   --ifm-color-primary-darkest: #5a007e;
+  --ifm-color-primary-medium: #975cd6;
   --ifm-color-primary-light: #bd83fb;
-  --ifm-color-primary-lighter: #bd83fb;
```
Comment (cmichi) [inline src/css/custom.css:15]: Accidental remove?

**Comment by cmichi** [comment]
Just as an FYI: [This comment](https://github.com/paritytech/ink-docs/pull/145#issuecomment-1423869427) explains how to have the faucet work despite `localhost`.

**Comment by cmichi** [comment]
Thank you, Sam! The color is fine.

You can follow the above linked comment to test it locally, you'll then see that the hash label is too big for the box:

screenshot-dev use ink_8080-2023 02 11-02_20_08

Could you display something like this instead:

>Funds have been sent!
>
>[Click here](https://rococo.subscan.io/extrinsic/0x9f1a3c9c0f8840433bfe3e24290929179ff279dca75c339ee6ed83bd09abaf33) to view the transaction.

**Review by agryaznov — APPROVED** [review]

**Comment by cmichi** [comment]
@DoubleOTheven I've merged, but please still fix the above things.

I've noticed that the font doesn't seem to load correctly (only tested in on Safari):

Bildschirm­foto 2023-02-13 um 12 25 04

**Pull request merged**